src/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard Web Page</title>
+</head>
+<body>
+    <header>
+        <h1>Leaderboard</h1>
+    </header>
+    <section>
+        <div class="container">
+            <div class="scores-container">
+                <div class="refresh">
+                 <h2>Recent scores</h2>
+                 <button id="refresh-btn">Refresh</button>
+                </div>
+                <ul class="list">
+                    <li>Name: 100</li>
+                    <li>Name: 20</li>
+                    <li>Name: 50</li>
+                    <li>Name: 78</li>
+                    <li>Name: 125</li>
+                    <li>Name: 77</li>
+                    <li>Name: 42</li>
+                </ul>
+            </div>
+                <form>
+                    <h2>Add your score</h2>
+                    <input type="text" id="name" placeholder="Your name">
+                    <input type="text" id="score" placeholder="Your score">
+                    <button id="form-btn">Submit</button>
+                </form>
+        </div>
+    </section>
+</body>
+</html>
```

Comment:
- [ ] Kindly match the given wireframe.

Your Implementation | Wireframe
:---:|:---:
![](https://i.imgur.com/7pn93fB.png) | ![](https://i.imgur.com/gyeXhWX.png)